src/main/java/frc/robot/commands/GoToPose.java

```diff
@@ -1,106 +1,106 @@
-package frc.robot.commands;
-
-import com.pathplanner.lib.PathPoint;
-import edu.wpi.first.math.geometry.Pose2d;
-import edu.wpi.first.wpilibj.DriverStation;
-import edu.wpi.first.wpilibj2.command.Command;
-import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
-import frc.robot.commands.auto.PathPlannerCommand;
-import frc.robot.constants.miscConstants.AutoConstants;
-import frc.robot.subsystems.DrivetrainImpl;
-
-import java.util.ArrayList;
-import java.util.List;
-import java.util.function.Supplier;
-
-/**
```

Comment:
Delete this if it's just going to be commented out. The main branch should stay clean

Reply:
working on pull request for uncommenting it now